src/web/components/RightColumn.tsx

```diff
@@ -10,18 +10,22 @@ const hideBelowDesktop = css`
 
     ${from.desktop} {
         /* above 980 */
+        height: 100%;
         display: block;
         flex-basis: 300px;
         flex-grow: 0;
         flex-shrink: 0;
     }
 `;
 
+const hideForIE = css`
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
Could you comment a link to an explanation of this line here for prosperity?

Reply:
👍 Added SO link (it's basically feature detection but that's about as much as I understand)

Reply:
👍 Added link to SO